build.gradle

```diff
@@ -17,14 +17,20 @@
 import org.apache.tools.ant.taskdefs.condition.Os
 
 plugins {
-    id "com.github.spotbugs" version "${githubSpotbugsVersion}"
-    id "com.github.johnrengelman.shadow" version "${githubJohnrengelmanShadowVersion}"
-    id "de.undercouch.download" version "${underCouchDownloadVersion}"
-    id "net.researchgate.release" version "${researchgateReleaseVersion}"
+    id "com.github.spotbugs-base"
+    id "com.github.johnrengelman.shadow"
+    id "de.undercouch.download"
+    id "net.researchgate.release"
 }
 
 description = 'Ballerina - Azure Cosmosdb'
 
+ext.ballerinaLangVersion = project.ballerinaLangVersion
+ext.azureCosmosVersion = project.azureCosmosVersion
+ext.nettyVersion = project.nettyVersion
+ext.reactorNettyVersion = project.reactorNettyVersion
+ext.micrometerVersion = project.micrometerVersion
+
```

Comment:
We don't need these assignments.

Reply:
Fixed with [2280283f9bc1a8ab83cff0aedcaff14e6b45e0ca](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/112/commits/2280283f9bc1a8ab83cff0aedcaff14e6b45e0ca)